sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -107,12 +110,35 @@ add_sycl_plugin(level_zero
     "${CMAKE_CURRENT_SOURCE_DIR}/pi_level_zero.hpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/usm_allocator.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/usm_allocator.hpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/tracing.cpp"
```

Comment:
I think we should include tracing.cpp into pi_level_zero.so only if (SYCL_ENABLE_XPTI_TRACING) is true.
Otherwise even release version of the library distributed to customers will contain unnecessary code which is used for debugging.

Reply:
Actually I think we should do the opposite, i.e. remove the SYCL_ENABLE_XPTI_TRACING and always include the tracing capabilities. The "sycl-prof" and "sycl-trace" tools are meant to be available to the end-user, and should always produce the traces. FWIW, I already removed the SYCL_PI_LEVEL_ZERO_ENABLE_TRACING (made it to "1" always). Do you agree?

Reply:
I see. Yes, I agree.